bot/exts/help_channels/_cog.py

```diff
@@ -387,7 +387,14 @@ async def move_to_dormant(self, channel: discord.TextChannel) -> None:
         )
 
         log.trace(f"Sending dormant message for #{channel} ({channel.id}).")
-        embed = discord.Embed(description=_message.DORMANT_MSG)
+        available_category = await channel_utils.try_get_channel(constants.Categories.help_available)
+        embed = discord.Embed(
+            description=_message.DORMANT_MSG.format(
+                dormant=channel.category.name,
```

Comment:
Testing this locally, it seems the channel object here isn't actually updated with the new category when it's moved on the lines above, so the embed reads as below. As you can see, it still references the in use category.

![image](https://user-images.githubusercontent.com/9753350/129018005-796e5bdc-0eee-4d6c-a246-39b1ad2d3db3.png)

Reply:
Fixed in [this](https://github.com/python-discord/bot/pull/1738/commits/99bfb15037ce7a6f88a403f8c1d2ab36d852874a) commit.